### ISSUE TITLE
wrong axis types for `lines`

### ISSUE THREAD

**Issue by fatteneder** [JuliaPlots/Makie.jl#1943]
# MWE

```julia
using GLMakie
GLMakie.activate!()

pts = fill(Point2f(1.0), 10)
lines(pt) # gives a 2D axis

pts = fill(Point2f(NaN), 10)
lines(pt) # gives a 3D axis
```

I would expect to receive a 2D axis in both cases.

**Comment by SimonDanisch** [comment]
>https://github.com/JuliaPlots/Makie.jl/pull/1770 is available with v0.17.1 and the error is fixed.

That PR got merged into [breaking-release](https://github.com/JuliaPlots/Makie.jl/pull/1918), so it's not tagged yet.
If you check out that branch, you will get a 2D axis.

**Comment by fatteneder** [comment]
Indeed. But then something else must have fixed the error message reported in #1669.

Also: Why does #1770 show up here as merged? https://github.com/JuliaPlots/Makie.jl/releases/tag/v0.17.1

**Comment by fatteneder** [comment]
Just tested with https://github.com/JuliaPlots/Makie.jl/pull/1918 and it is indeed fixed there. Thanks.
